lib/jormungandr/src/Cardano/Wallet/Jormungandr.hs

```diff
@@ -134,7 +137,12 @@ serveWallet (cfg, tr) databaseDir hostPref listen lj beforeMainLoop = do
             let (_, bp) = staticBlockchainParameters nl
             rndApi <- apiLayer tr (toWLBlock <$> nl)
             seqApi <- apiLayer tr (toWLBlock <$> nl)
-            startServer tr nPort bp rndApi seqApi
+
+            -- StakePool
+            PoolSqlite.withDBLayer cfg tr (poolDBPath databaseDir) $
+                \splDB -> do
```

Comment:
why the line-break here :grimacing: ?

Reply:
Because we have a rule saying lines must be 80 characters or less 😬 This one would have been 81.

Maybe importing `Cardano.Pool.DB.Sqlite` as `Pool` instead of `PoolSqlite` would actually be nice though…

Reply:
Hmm. Yes. But the guidelines are... Guidelines. When it makes sense, we can break them and we sometimes do.